packages/cli/src/commands/replay/replay.command.ts

```diff
@@ -1,5 +1,9 @@
 import { mkdir, mkdtemp, writeFile } from "fs/promises";
```

Comment:
The diff in this file is a bit tricky to read because a bunch of code has been moved around. May be easier to compare before vs after.